### ISSUE TITLE
TSV:  Fewer scroll bars on the right please!

### ISSUE THREAD

**Issue by SRGDamia1** [ODM2/ODM2DataSharingPortal#584]
The page and plot should be dynamically sized to not need scroll bars.  One scroll bar should be enough.  I'm getting three on staging right now.

![image](https://user-images.githubusercontent.com/5374350/163267909-760e3ed3-ce45-4b00-bf96-2c21af8204da.png)

**Comment by ptomasula** [comment]
Good catch. Yeah I think three scrolls bars looks terrible. The best I can get it down to in the short term is two, which is now live on staging. 

The outer scroll bar controls the page as a whole and is needed due to the size of the footer. The inner scroll for the TSV control panel. We could revisit this for a future release and consider options like removing the footer. We could also look at adding in [accordions](https://jqueryui.com/accordion/) to make better use of the real estate in the TSV control panel.
